docker/hoodie/hadoop/prestobase/Dockerfile

```diff
@@ -0,0 +1,69 @@
+## Presto docker setup is based on https://github.com/smizy/docker-presto
+
+ARG HADOOP_VERSION=2.8.4
+ARG HIVE_VERSION=2.3.3
+FROM apachehudi/hudi-hadoop_${HADOOP_VERSION}-base:latest as hadoop-base
+
+ARG PRESTO_VERSION=0.217
+
+ENV PRESTO_VERSION       ${PRESTO_VERSION}
+ENV PRESTO_HOME          /opt/presto-server-${PRESTO_VERSION}
+ENV PRESTO_CONF_DIR      ${PRESTO_HOME}/etc
+ENV PRESTO_LOG_DIR       /var/log/presto
+ENV PRESTO_JVM_MAX_HEAP  16G
```

Comment:
should these be lower? the memory limits seem too high for local machine

Reply:
+1

Reply:
Reduced the memory sizes